CAM-10103 Updated and split Kubernetes docs (#2668)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Deploying Anywhere/kubernetes.md

### Overview

With the Kubernetes provider of Cloud Application Manager, you can create and manage any Kubernetes resources, using template boxes employing declarative YAML templates.

This article explains how to set up a Kubernetes provider, as well as how to create a service account in the Kubernetes cluster to use with Cloud Application Manager.

> For template boxes, policies and deployments, please visit the [second part of this article](../Automating%20Deployments/using-kubernetes.md).

Reviewer: ```suggestion
For how to create Kubernetes policies, template boxes and deployments, please visit the [second part of this article](../Automating%20Deployments/using-kubernetes.md).
```

### Prerequisites

- An active Cloud Application Manager account

> - Access and *cluster-admin* privileges to a Kubernetes cluster

Reviewer: ```suggestion
- Access with *cluster-admin* privileges to a Kubernetes cluster
```

### Getting started with Kubernetes on Cloud Application Manager

The implementation of Kubernetes in Cloud Application Manager was done in a way that it shall work with any modern Kubernetes cluster, having a recent version of Kubernetes installed. This includes the mayor cloud providers, like Amazon EKS, AKS on Microsoft Azure and GKS on Google Cloud.

> Cloud Application Manager is capable of discovering supported APIs, their versions, and supported resource types withing configured Kubernetes cluster.

Reviewer: ```suggestion
Cloud Application Manager is capable of discovering supported APIs, their versions, and supported resource types within the configured Kubernetes cluster.
```

It is recommended to create a separate ServiceAccount for Cloud Application Manager to interact with the Kubernetes cluster. This process requires a *kubectl* environment installed and configured. There is an [excellent guide](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for that.

### Configuring kubectl

#### Setting up kubectl for Amazon EKS

> Provided that you already have the [AWS Command Line Interface](https://aws.amazon.com/cli/) installed, execute the following command in the shell to configure *kubectl* with the credentials of the EKS cluster. (Replace *\$REGION* and *\$CLUSTER_NAME* accordingly.)

Reviewer: ```suggestion
Considering that you already have the [AWS Command Line Interface](https://aws.amazon.com/cli/) installed, execute the following command in the shell to configure *kubectl* with the credentials of the EKS cluster. (Replace *\$REGION* and *\$CLUSTER_NAME* accordingly.)
```

```
az aks get-credentials --resource-group $RESOURCE_GROUP_NAME --name $CLUSTER_NAME
```

Alternatively, you can select *Kubernetes Service* on Azure Portal, select the cluster, and navigate to the *Overview* page, and under *View Kubernetes Dashboard* click *View Connection Steps*, copy step 3 and execute in Cloud Shell.

#### Setting up kubectl for Google GKS

Open *Cloud Shell* and execute the following commands (replacing *\$CLUSTER_NAME* and *\$ZONE* accordingly):

```
gcloud container clusters list
gcloud container clusters get-credentials $CLUSTER_NAME --zone $ZONE
```

### Create a Service Account in Kubernetes

The following steps are the same for any kind of Kubernetes clusters. *kubectl* has to be set up for the cluster to execute these commands. Execute these command snippets from below in the same shell and in order, (because they rely on the state stored in shell variables). It is safe to execute the commands multiple times, however `kubectl create` prints an error if the resource already exists.

> First to check whether *kubectl* is properly configured, execute:

Reviewer: ```suggestion
First, to check if *kubectl* is properly configured you can execute:
```
